frontend/pages/joinUs.js

```diff
@@ -0,0 +1,203 @@
+import React, { useEffect, useState } from "react";
+import PageHeader from "../components/header";
+import styles from "../styles/joinUs.module.css";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import Slideshow from "../components/QuotesSlideshow.js";
+import NewsletterSection from "../components/NewsletterSection";
+import { BootstrapTooltip } from "../components/BootstrapTooltip";
+import LoadingScreen from "../components/LoadingScreen";
+import { memberQuotes } from "../data/joinData";
+import { socials } from "../data/joinData";
+import SimpleAccordion from "../components/joinUsAccordion";
+import PortfolioGrid from "../components/joinUsGrid";
+import Image from "next/image";
+
+const JoinUs = () => {
+  const [hideSpinner, setHideSpinner] = useState(false);
+  const [openNewsletter, setOpenNewsletter] = useState(false);
+  const [sourceLoading, setSourceLoading] = useState(true);
+  const [headerLoading, setHeaderLoading] = useState(true);
+
+  // close newsletter
+  const callbackModal = () => {
+    setOpenNewsletter(false);
+  };
+
+  // start at the top of the page
+  useEffect(() => {
+    window.scrollTo(0, 0);
+    setTimeout(() => {
+      setSourceLoading(false);
+    }, 10000);
+  }, []);
+
+  return (
+    <div>
+      {sourceLoading && !hideSpinner && headerLoading ? (
+        <LoadingScreen />
+      ) : (
+        <>
+          <PageHeader
+            imgUrl="/headers/join-header-2.jpg"
+            title="Join Us"
+            imageLoading={setHeaderLoading}
+          />
+          {/* Start of connect with us */}
+          <h2 className={styles.header}>Connect with us</h2>
+          <div className={styles.joinUsBody}>
+            <div className={styles.joinUsRight}>
+              <div className={styles.joinButtons}>
+                {Object.keys(socials).map((social, index) => {
+                  return (
+                    <BootstrapTooltip
+                      key={social}
+                      title={
+                        <>
+                          <div className="tooltipTitle">{social}</div>
+                        </>
+                      }
+                    >
+                      <a
+                        className={styles.joinLink}
+                        key={index}
+                        href={socials[social][0]}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        <Image
+                          src={`/icons/${socials[social][1]}`}
+                          alt={social}
+                          height="250px"
+                          width="250px"
+                        />
+                      </a>
+                    </BootstrapTooltip>
+                  );
+                })}
+              </div>
+            </div>
+            <div className={styles.googleMap}>
```

Comment:
Dimensions of the google map should match the website. Open local instance on one tab and have the wit website on another tab to compare. 

Submitted:
![image](https://user-images.githubusercontent.com/80564580/174699541-a02debd9-410a-4532-90af-09c36f0893c2.png)

What is on the website: 
![image](https://user-images.githubusercontent.com/80564580/174699566-7fed5ea8-83e9-4b9d-95d6-7c89fc55d020.png)